asm/macros/event.inc

```diff
@@ -2338,3 +2338,21 @@
 	.macro hideitemdescription
 	callnative ScriptHideItemDescription
 	.endm
+
+   .macro increasedifficulty
+    callnative Script_IncreaseDifficulty
+   .endm
+
+    .macro decreasedifficulty
+    callnative Script_DecreaseDifficulty
+    .endm
+
+	.macro getdifficulty var:req
+	.byte 0xe5
+	.2byte \var
+	.endm
+
+	.macro setdifficulty difficulty:req
+	.byte 0xe6
+	.2byte \difficulty
```

Comment:
Is there a rationale to using opcodes here instead of `callnative` / `special` like above?

Reply:
nope

[`b178d44` (#5337)](https://github.com/rh-hideout/pokeemerald-expansion/pull/5337/commits/b178d44e6c9113ef3d43c2b6b1462a3598d22ca1)